appveyor-linux-binary.yml

```diff
@@ -52,7 +52,7 @@ install:
   - sh: "docker info"
   - sh: "docker version"
   - sh: "nvm install ${NODE_VERSION}"
-  - sh: "npm install npm@7.24.2 -g"
+  - sh: "npm install npm@10.2.3 -g"
```

Comment:
Nodejs 18.18.2 comes with `npm==9.8.1` but I just installed the latest anyways. This shouldn't affect anything in our tests, but we can remove this line if we wanted to stuck with the provided version instead. We need `>=8.0.0`, so either way works.

https://nodejs.org/en/about/previous-releases#looking-for-latest-release-of-a-version-branch